pkg/operator/certrotationcontroller/internalloadbalancer.go

```diff
@@ -18,7 +18,6 @@ func (c *CertRotationController) syncInternalLoadBalancerHostnames() error {
 	hostname := infrastructureConfig.Status.APIServerURL
 	hostname = strings.Replace(hostname, "https://", "", 1)
 	hostname = hostname[0:strings.LastIndex(hostname, ":")]
-	hostname = strings.Replace(hostname, "api.", "api-int.", 1)
```

Comment:
we shouldn't remove this until https://github.com/openshift/installer/pull/1633 merges right?